src/API/Data/PeopleRepository.cs

```diff
@@ -68,6 +68,28 @@ private static IQueryable<Person> GetPeopleFilteredByArea(PeopleContext db, Peop
                             )
                             SELECT id FROM parentage
                             WHERE root_id <> 1))");
+        
+        private static IQueryable<Person> SearchHrPeopleByNetId(PeopleContext db, HrPeopleSearchParameters query)
```

Comment:
Do we want to rename `SearchHrPeopleByNetId()` and `SearchBothByNetId()` since they aren't actually limitted to just netId?

Reply:
Yeah -- something like `...ByNameOrNetId`?

Reply:
Renamed in e680153